airbyte-webapp-e2e-tests/cypress/integration/connection.spec.ts

```diff
@@ -121,6 +127,123 @@ describe("Connection main actions", () => {
     deleteDestination(destName);
   });
 
+  it("creates a connection, then edits the schedule type", () => {
+    const sourceName = appendRandomString("Test connection source cypress PokeAPI");
+    const destName = appendRandomString("Test connection destination cypress");
+
+    createTestConnection(sourceName, destName);
+
+    cy.get("div").contains(sourceName).should("exist");
+    cy.get("div").contains(destName).should("exist");
+
+    openSourceDestinationFromGrid(sourceName);
+
+    goToReplicationTab();
+
+    selectSchedule("Cron");
+    submitButtonClick();
+    checkSuccessResult();
+
+    selectSchedule("Manual");
+    submitButtonClick();
+    checkSuccessResult();
+
+    selectSchedule("Every hour");
+    submitButtonClick();
+    checkSuccessResult();
+
+    deleteSource(sourceName);
+    deleteDestination(destName);
+  });
+
+  // Analytics calls are not made in CI.
+  it.skip("Should make analytics calls", () => {
```

Comment:
should we remove this then?

Reply:
Maybe! @timroes is there a way we can enable them in CI? Or do you think we should just toss this test I added?